src/test/java/org/broadinstitute/consent/http/db/VoteDAOTest.java

```diff
@@ -43,15 +41,19 @@ void testFindVoteById() {
     User user = createUserWithRole(UserRoles.CHAIRPERSON.getRoleId());
     Dataset dataset = createDataset();
     String darCode = "DAR-1234567890";
-    Integer collection_id = darCollectionDAO.insertDarCollection(darCode, user.getUserId(),
+    Integer collectionId = darCollectionDAO.insertDarCollection(darCode, user.getUserId(),
         new Date());
-    DataAccessRequest dar = createDataAccessRequestWithDatasetAndCollectionInfo(collection_id,
+    DataAccessRequest dar = createDataAccessRequestWithDatasetAndCollectionInfo(collectionId,
         dataset.getDatasetId(), user.getUserId());
     Election election = createDataAccessElection(dar.getReferenceId(), dataset.getDatasetId());
     Vote vote = createDacVote(user.getUserId(), election.getElectionId());
 
     Vote foundVote = voteDAO.findVoteById(vote.getVoteId());
     assertNotNull(foundVote);
+    assertNotNull(foundVote.getCreateDate());
+    assertEquals(user.getUserId(), foundVote.getUserId());
+    assertEquals(election.getElectionId(), foundVote.getElectionId());
+    assertEquals(VoteType.DAC.getValue(), foundVote.getType());
```

Comment:
Expanded this test to ensure the new insert is populating create date. Note that line 49 (`createDacVote`) calls a helper method that exercises the modified insert statement.